### ISSUE TITLE
"Find" feature in mobile Safari does not work as expected

### ISSUE THREAD

**Issue by elie195** [ampproject/amphtml#5750]
"Find" feature in mobile Safari does not work as expected.
## How do we reproduce the issue?

https://www.google.com/amp/appleinsider.com/articles/16/09/09/apples-wide-color-screen-on-the-iphone-7-will-lead-to-more-faithful-color-reproduction/amp/?client=safari
1. Enter "true tone" in Safari address bar and select "On This Page (2 matches)"
2. Press down/up arrows a few times

Expected: Match will be highlighted and scrolled to.
Actual: Nothing happens except the "find" window changing from "1 of 2" to "2 of 2" and vice-versa.
## What browsers are affected?

Reproduced on iPhone with iOS 10.1 beta 5
## Which AMP version is affected?

Not sure, couldn't get page to load in a desktop browser in order to check the console.

**Comment by jridgewell** [comment]
/to @dvoytenko 

This might be fixed by the new iOS Viewport Binding.

**Comment by dvoytenko** [comment]
This is a known issue for iOS Safari. I thought I saw the bug before, but couldn't find it now. So I created a new one here: https://bugs.webkit.org/show_bug.cgi?id=163911

**Comment by adelinamart** [comment]
One referenced issue was fixed. Can we check if this is still an issue? Thanks.

**Comment by cramforce** [comment]
We are waiting on https://bugs.webkit.org/show_bug.cgi?id=163911

**Comment by adelinamart** [comment]
It seems there's some new activity on https://bugs.webkit.org/show_bug.cgi?id=163911
